CHANGELOG.md

```diff
@@ -9,14 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### :sparkles: Release Note <!-- optional -->
 
 ### :rocket: Enhancements
-- ...
+- [#2026](https://github.com/reviewdog/reviewdog/pull/2026) Add reporter for GitHub annotations `github-annotations`
```

Comment:
Can you briefly mention it's the same as `github-pr-annotations`?
Ideally, we can also mention `github-pr-annoations` are deprecated.

Reply:
Done